include/wx/stringops.h

```diff
@@ -44,9 +44,19 @@ struct WXDLLIMPEXP_BASE wxStringOperationsWchar
     static ptrdiff_t DiffIters(const Iterator& i1, const Iterator& i2)
         { return i1 - i2; }
 
+#if wxUSE_UNICODE_WCHAR && SIZEOF_WCHAR_T == 2
+    struct Utf16CharBuffer
+    {
+        wchar_t data[3];
```

Comment:
I'd add a default ctor initializing `data` to zeros, you currently do it explicitly in `EncodeChar()`, of course, but it's safer to not allow creating objects with uninitialized data in the first place.